Source/AwsCommonRuntimeKit/sdkutils/FileBasedConfigSectionType.swift

```diff
@@ -0,0 +1,18 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0.
+import AwsCSdkUtils
+
+/// Type of section in a config file
+public enum FileBasedConfigSectionType {
+    case profile
+    case sso_session
+}
```

Comment:
If we need to add more types in the future, it will be a breaking change. @epau, do you have any ideas on how we can improve this?

Reply:
Imo this is ok.
Consumers can control if this is a breaking change or not by either being exhaustive (if they want to break when a new case is added) or by using `default` (if they don't want to break).